Sources/ExtensionDictionary.swift

```diff
@@ -24,8 +24,50 @@
 //
 
 import Foundation
-
-internal extension Dictionary {
+public extension Dictionary {
```

Comment:
i don't think we need to mark the entire extension `public` - lets just remove that keyword here and mark the individual functions with their access modifiers